src/plugins/interactive_setup/server/kibana_config_writer.test.ts

```diff
@@ -131,7 +132,7 @@ describe('KibanaConfigWriter', () => {
         elasticsearch.hosts: [some-host]
         elasticsearch.serviceAccountToken: some-value
         elasticsearch.ssl.certificateAuthorities: [/data/ca_1234.crt]
-        xpack.fleet.outputs: [{id: fleet-default-output, name: default, is_default: true, is_default_monitoring: true, type: elasticsearch, hosts: [some-host], ca_sha256: fingerprint256}]
+        xpack.fleet.outputs: [{id: fleet-default-output, name: default, is_default: true, is_default_monitoring: true, type: elasticsearch, hosts: [some-host], ca_trusted_fingerprint: d486ce00ac71e41d2b70d087a555fa5dd1936cdb458079537ba3ac133e4834d6}]
```

Comment:
nit: Much prefer this to the previous config name but is it worth aligning this with elasticsearch-js client for consistency? 
https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/current/basic-config.html

> `caFingerprint` - If configured, verify that the fingerprint of the CA certificate that has signed the certificate of the server matches the supplied fingerprint. Only accepts SHA256 digest fingerprints.

(We don't use `elasticsearch.ssl.trustedCertificateAuthorities` either but I don't know what Fleet conventions are)

Reply:
The field is named `ca_trusted_fingerprint` in libbeat and the agent, so I think we probably to keep that consistent here.